apps/src/dance/lab2/views/GenerateDancer.tsx

```diff
@@ -442,6 +446,11 @@ const GenerateDancer: React.FunctionComponent<DancerGenerateProps> = ({
                 />
               </>
             )}
+          {isReadOnly && (
+            <BodyThreeText className={moduleStyles.readOnlyText}>
+              <EmText>{`AI generated a dancer based on this prompt:`}</EmText>
+            </BodyThreeText>
```

Comment:
I would just put the text in a `<div>` to get the same styling as other similar text in the guide.